assembl/auth/util.py

```diff
@@ -447,29 +454,33 @@ def add_user(name, email, password, role, force=False, username=None,
                 preferred_email=email,
                 verified=True,
                 password=password,
-                creation_date=datetime.utcnow())
+                creation_date=datetime.utcnow(),
+                is_machine=kwargs.get('is_machine', False))
             db.add(user)
         if username:
             db.add(Username(username=username, user=user))
+
     for account in user.accounts:
         if isinstance(account, EmailAccount) and account.email_ci == email:
             account.verified = True
             account.preferred = True
             break
-    else:
-        account = EmailAccount(
-            profile=user,
-            email=email,
-            preferred=True,
-            verified=True)
-        db.add(account)
+        else:
+            account = EmailAccount(
+                profile=user,
+                email=email,
+                preferred=True,
+                verified=True)
+            db.add(account)
+
```

Comment:
@amensouissi why did you change this? The else was associated to the for, if no user.accounts, it should execute the else.